build/task-build-browser.js

```diff
@@ -35,6 +41,20 @@ async function buildBrowser() {
   return await webpackBuild(development ? browserDevConfig : browserProdConfig);
 }
 
+async function buildBrowserAlt() {
+  const { SRC_DIR, SHARED_DIR } = BrowserAltDirs;
+  const id = 'browser-alt';
+
+  if (!(await shouldRebuild(id, [SRC_DIR, id], [SHARED_DIR, 'ui/shared']))) {
+    logger.info(colors.green(`No changes in ${id}.`));
+    return { close: () => {} };
+  }
+
+  logger.info(colors.cyan(`Building ${id}...`));
+  const { development } = getBuildConfig();
+  return await webpackBuild(development ? browserAltDevConfig : browserAltProdConfig);
+}
```

Comment:
On the other hand, all of this is yucky to have to write for every single frontend.